### DISCUSSION TITLE
Change Premium logo to BC

### DISCUSSION THREAD

**Discussion by ShadowDG3** [tersiswilvin/Roblox-Legacy-Old-Theme#10] [Ideas]
Change the premium logo to BC.

$4.99 = BC
$9.99 = TBC
£19.99 = OBC

Also, can you add the 2015 catalog?

**Comment by tersiswilvin** [comment]
Hi [ShadowDG3](https://github.com/ShadowDG3), premium to BC is in the works however I do not know when I will be able to add it as I am more focused on other things, the 2015/2016 catalog is planned however I do not have any plans on it at this time as it will likely be under the legacy pages options whenever I start on it.

**Reply by ShadowDG3** [reply]
Ok, thanks for the reply.

**Reply by ShadowDG3** [reply]
https://codepen.io/notakenusernamedefinetley/pen/JjLyzyW?editors=1100 Could this help?

**Comment by ghost** [comment]
What would be nice was if the whole old BC page was recreated, but like with my old group suggestion, I mentioned that I would imagine it would be really hard to recreate as the whole HTML/CSS has changed, same with the old Develop page.
